templates/single-product/short-description.php

```diff
@@ -26,7 +26,4 @@
 	return;
 }
 
-?>
-<div itemprop="description">
-	<?php echo apply_filters( 'woocommerce_short_description', $post->post_excerpt ) ?>
-</div>
+echo apply_filters( 'woocommerce_short_description', $post->post_excerpt );
```

Comment:
I think we should still have a wrapping div for the short description, for styling purposes. `.woocommerce-product-details__short-description`.